Makefile

```diff
@@ -27,36 +51,48 @@ endif
 
 test: clean up
 
-example: custom_ca
+format:
+	./format.sh
```

Comment:
I think pre-commit target solve that also
```suggestion
```

Reply:
Yes, `make pre-commit` does the same thing